charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -45,6 +45,10 @@ metadata:
   namespace: {{ .Values.rhdhOperator.subscription.namespace }}
 stringData:
   BACKEND_SECRET: {{ randAlphaNum 24 | nospace | b64enc }}
+{{- if ne .Values.rhdhOperator.k8s.clusterToken "" }}
+  K8S_CLUSTER_URL: https://api.{{ include "cluster.domain" . }}:6443
```

Comment:
this is being resolved to `https://api.apps.CLUSTER_DOMAIN:6443` which causes it to fail. `apps` should be removed. The link should point to the API Server (e.g. by `oc cluster-info`).